orcid-web/src/main/resources/orcid-frontend-security.xml

```diff
@@ -130,6 +131,23 @@
 	<sec:http pattern="/static/**" security="none" />
 	<sec:http request-matcher="regex" pattern="/(\d{4}-){3,}\d{3}[\dX](\?.*)?" security="none" create-session="stateless" />
 	<sec:http request-matcher="regex" pattern="/(\d{4}-){3,}\d{3}[\dX]/print" security="none" create-session="stateless" />
+	
+	<sec:http pattern="/togglz/.*" 
+		use-expressions="false" 
+		access-decision-manager-ref="accessDecisionManager" 
+		authentication-manager-ref="authenticationManager" 
+		create-session="ifRequired"
+		request-matcher="regex" >
+		<sec:csrf disabled="true"/>
+        <sec:anonymous enabled="false" />        
+        <sec:intercept-url pattern="/togglz/index" access="ROLE_ADMIN" />        
```

Comment:
/togglz/* would be ok, but, there are only two endpoints that we need /index and /edit, and, we can configure different security for each, so, some users can see and some other users can edit, so, I just did that to highlight we could do that in a future